org.lamport.tla.toolbox.tool.tlc.ui/src/org/lamport/tla/toolbox/tool/tlc/ui/editor/page/ResultPage.java

```diff
@@ -1027,6 +1027,7 @@ public static void createTableColumns(Table stateTable, ResultPage page)
             for (int i = 0; i < columnTitles.length; i++)
             {
                 TableColumn column = new TableColumn(stateTable, SWT.NULL);
+                column.setAlignment(SWT.RIGHT);

```

Comment:
@quaeler Does this alignment conflict with your recent UI styling?

Reply:
Nope - it should not.